README.md

```diff
@@ -10,4 +10,5 @@ How to start the DropwizardWebService application
 Health Check
 ---
 
-To see your applications health enter url `http://localhost:8081/healthcheck`
\ No newline at end of file
+To see your applications health enter url `http://localhost:8081/healthcheck`
```

Comment:
Can you add how to run lint to the Readme

Reply:
change made